tests/api/test_recents.py

```diff
@@ -77,7 +83,9 @@ def test_get_recent_nodes(client):
                     'epoch': None,
                     'version': None,
                     'release': None,
-                    'id': '44444'
+                    'id': '44444',
+                    'display_name': 'None-None-None',
```

Comment:
Can you add `name`, `version`, and `release` to the `KojiBuild` you create so it's not `None-None-None`?